tsconfig.json

```diff
@@ -18,7 +18,7 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "noEmit": true,
-    "jsx": "react-jsx"
+    "jsx": "preserve"
```

Comment:
IDK what or why this was changed.

Reply:
maybe cuz u havent really written any jsx?? idk